### ISSUE TITLE
the problem of -u(--abnormalUnmapFlag from wrapper) 

### ISSUE THREAD

**Issue by snowylxx** [liulab-dfci/TRUST4#215]
So I run the TRUST4, here was a error that cause there were not outfiles. 
error: Two reads from the unaligned fragment are not showing up together. Please use -u(--abnormalUnmapFlag from wrapper) option.
I see the bam file is pair-end. how to handle this?

**Comment by mourisl** [comment]
You just need to add the "--abnormalUnmapFlag" option when using the "run-trust4" command.

**Comment by snowylxx** [comment]
oh ,thanks!

**Comment by snowylxx** [comment]
so I add the option but there were another error occurred:
/software/TRUST4-1.0.12/bam-extractor: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by /software/TRUST4-1.0.12/bam-extractor: /software/TRUST4-1.0.12/bam-extractor)
/software/TRUST4-1.0.12/bam-extractor: /lib64/libstdc++.so.6: version `CXXABI_1.3.8' not found (required by /software/TRUST4-1.0.12/bam-extractor)
/software/TRUST4-1.0.12/bam-extractor: /lib64/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by /software/TRUST4-1.0.12/bam-extractor)
/software/TRUST4-1.0.12/bam-extractor:  /lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /software/TRUST4-1.0.12/bam-extractor)
/software/TRUST4-1.0.12/bam-extractor:  /lib64/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by /software/TRUST4-1.0.12/bam-extractor)

**Comment by mourisl** [comment]
How did you install TRUST4?
This is strange. You should receive this error in your first run. Adding the "--abnormalUnmapFlag" should not create libstdc++ issue.

**Comment by snowylxx** [comment]
I just need to log out and rerun the program again.

But I have one more question. I will run multiple bam files at once, and some will prompt: Two reads from the unaligned fragment are not showing up together Please use - u (-- abnormalUnmapFlag from wrapper) option. Some can be run directly. May I add the option '-- abnormalUnmapFlag' for all files for convenience?

**Comment by mourisl** [comment]
Yes, you may add the option for all files. It should not affect the results.
